<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ongletCalcul</class>
 <widget class="QMainWindow" name="ongletCalcul">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/Run.png</normaloff>img/Run.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background-color:rgb(255, 255, 255);
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="2">
         <widget class="QPushButton" name="parcourirRes">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="pathObs"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="parcourirObs">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_66">
          <property name="text">
           <string>Observations</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="resDirPath"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_69">
          <property name="text">
           <string>Dossier des résultats</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="parcourirPoints">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_67">
          <property name="text">
           <string>Points</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_68">
          <property name="text">
           <string>Paramètres</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="pathPoints"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="pathParam"/>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="parcourirParam">
          <property name="text">
           <string>Parcourir</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="2" rowspan="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="runCalcul">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 218, 160);
border-color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBoxCtrlCoh">
          <property name="text">
           <string>Contrôles de cohérence</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
